### PR TITLE
DEV-2

### PR DESCRIPTION
**Pull request by ElizavetaKovalchuk**

### CODE REVIEW

File: EntryPoint.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Text;
+
+namespace DEV_2
+{
+    class EntryPoin
```
Comment (IgorKheidorov) [inline EntryPoint.cs]: grammar error

File: EntryPoint.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        {
+            Console.WriteLine("Enter a string:");
+            string line = Console.ReadLine();
+            if (line.Length == 0)
+            {
+                Console.WriteLine("String is empty");
+            }
+            else
+            {
+                SerchOddElements enteredSymbol = new SerchOddElements(line);
```
Comment (IgorKheidorov) [inline EntryPoint.cs]: bad value name enteredSymbol

File: EntryPoint.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                Console.WriteLine(enteredSymbol.SerchOddSymbol());
+                Console.ReadKey();
```
Comment (IgorKheidorov) [inline EntryPoint.cs]: Never add ReadKey() in the end of the application, what for?

File: SerchOddElements.cs [hunk 4]
```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Text;
+namespace DEV_2
+{
+    class SerchOddElements
```
Comment (IgorKheidorov) [inline SerchOddElements.cs]: Bad class name, no comments, spelling error

File: SerchOddElements.cs [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+{
+    class SerchOddElements
+    {
+        public string enteredString;
+        public SerchOddElements(string enteredSymbol)
+        {
+            enteredString = enteredSymbol;
+        }
+        /// <summary>
+        /// Serch elements with odd ideks in a string
```
Comment (IgorKheidorov) [inline SerchOddElements.cs]: The method not only search but create string, but this is not obvious from the name

Comment (IgorKheidorov) [inline SerchOddElements.cs]: ideks - what is it?

File: SerchOddElements.cs [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+        /// </summary>
+        /// <returns> string of elements with odd indeks</returns>
+        public string SerchOddSymbol()
+        {
+            StringBuilder oddElements = new StringBuilder(enteredString);
+            for (int i = 1; i < oddElements.Length; i++)
+            {
+                oddElements = oddElements.Remove(i,1);
+            }
+            string newString = oddElements.ToString();
```
Comment (IgorKheidorov) [inline SerchOddElements.cs]: simpler return oddElements.ToString();

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]
Give normal name for pull request

File: Creator.cs [hunk 7]
```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Text;
+namespace DEV_2
```
Comment (AlexanderSoroka) [inline Creator.cs]: It's better to have blank line before namespace

File: Creator.cs [hunk 8]
```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Text;
+namespace DEV_2
+{
+    /// <summary>
+    /// Serch elements with odd indeks in a string and create string
```
Comment (AlexanderSoroka) [inline Creator.cs]: Serch, indeks - moya tvoya pochti ponyala :)

File: Creator.cs [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+    /// </summary>
+    class Creator
+    {
+        public string sequnceOfSymbol;
+        public Creator(string sequnceOfSymbol)
+        {
+            sequnceOfSymbol = enteredSequence;
+        }
+        /// <summary>
```
Comment (AlexanderSoroka) [inline Creator.cs]: add blank line before comment

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

**Pull request merged**